README.md

```diff
@@ -241,17 +267,112 @@ Refer to the following README sections on push setup:
 
 ### Collecting Push Tokens
 
-Push tokens must be collected in the native layer. Follow the platform-specific instructions below:
+Push tokens must be collected both in your app's react native code or in the native code. Below sections discuss both approaches, and
+you are free to pick one that best suits your app's architecture. Note that doing this in one location is sufficient.
+
+#### React Native Token Collection
+
+In order to collect the APNs push token in your React Native code you need to,
```

Comment:
So just to be clear we are recommending adding a firebase library to collect APNs tokens. We think this is ok since they will probably already be using this library right?

Reply:
```suggestion
In order to collect the APNs push token in your React Native code you need to:
```